user_docs/en/changes.md

```diff
@@ -43,6 +43,7 @@ The available options are:
   * It is now possible to use braille display routing keys to move the text cursor. (#9101)
   * It is now possible to use the review cursor selection commands to select text.
 * Updating NVDA while add-on updates are pending no longer results in the add-on being removed. (#16837)
+* Correct internal gesture ID for Seika Notetaker. (#17047, @school510587)
```

Comment:
This needs to be moved to 2025.1. It also needs to explain the user-facing change.